scripts/run-ethspecify.js

```diff
@@ -0,0 +1,145 @@
+ * 3. Runs ethspecify on this HTML file
+ * 4. Processes and displays the results
+ * 5. Cleans up the temporary file
+ */
+
+const fs = require('fs');
+const { execSync } = require('child_process');
+const path = require('path');
+
+// Import the spec references
+// Note: Need to transpile TypeScript to JS first or use a runtime like ts-node
+// For simplicity, we'll read the file and extract the data
+const specReferencesPath = path.join(__dirname, '../configs/spec-references.ts');
+
+// Read and parse the spec references file
+const specRefsContent = fs.readFileSync(specReferencesPath, 'utf8');
+const specRefsMatch = specRefsContent.match(/export const SpecReferences\s*=\s*(\[[\s\S]*?\n\]);/);
+if (!specRefsMatch) {
+  console.error('Could not find SpecReferences array in spec-references.ts');
+  process.exit(1);
+}
+
+// Create a temporary directory for ethspecify to process
+const tempDir = path.join(__dirname, 'temp-ethspecify');
+if (!fs.existsSync(tempDir)) {
+  fs.mkdirSync(tempDir);
+}
+
+// Create a temporary HTML file with all the spec tags
+const tempHtmlPath = path.join(tempDir, 'spec-references.html');
+let htmlContent = `
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Lodestar Spec References</title>
+</head>
+<body>
+  <h1>Ethereum Specification References for Lodestar</h1>
+`;
+
+// Parse the SpecReferences array content using regex to extract each entry
+const arrayString = specRefsMatch[1].replace(/\/\*[\s\S]*?\*\/|\/\/.*/g, '');
+const specRefEntries = [];
+const entryRegex = /{[^}]*component:\s*"([^"]+)"[^}]*filePath:\s*"([^"]+)"[^}]*specTag:\s*`([^`]+)`[^}]*}/g;
+let match;
+
+while ((match = entryRegex.exec(arrayString)) !== null) {
+  specRefEntries.push({
+    component: match[1],
+    filePath: match[2],
+    specTag: match[3]
+  });
+}
+
+// Add validation for @spec tags
+const missingSpecTags = [];
+specRefEntries.forEach(ref => {
+  if (!specRefsContent.includes(`@spec ${ref.component}`)) {
+    missingSpecTags.push(ref.component);
+  }
+});
+
+if (missingSpecTags.length > 0) {
+  console.log('\nWarning: Missing @spec tags for:');
+  missingSpecTags.forEach(component => console.log(`- ${component}`));
+}
+
+// Add each spec reference to the HTML content
+specRefEntries.forEach(ref => {
+  htmlContent += `
+  <div class="spec-reference">
+    <h3>${ref.component}</h3>
+    <p>File: ${ref.filePath}</p>
+    <pre>
+      /**
+       * ${ref.specTag}
+       */
+    </pre>
+  </div>
+  `;
+});
+
+htmlContent += `
+</body>
+</html>
+`;
+
+// Write the HTML content to the temporary file
+fs.writeFileSync(tempHtmlPath, htmlContent);
+console.log(`Created temporary HTML file with ${specRefEntries.length} spec references`);
+
+// Run ethspecify on the temporary file
+try {
+  console.log('Running ethspecify...');
+  // Make sure the virtual environment is activated if needed
+  const ethspecifyCommand = `source ethspecify_env/bin/activate && ethspecify process --path "${tempDir}"`;
```

Comment:
Sorry for one more ask. 

Now that EthSpecify supports specifying a spec version, can you pin the version to `v1.6.0-alpha.1` in `run-ethspecify.js`?